index.js

```diff
@@ -274,9 +282,11 @@ function processJSXElement(annotateFragments, t, jsxElement, componentName, sour
 function functionBodyPushAttributes(annotateFragments, t, path, componentName, sourceFileName, attributeNames, ignoreComponentsFromOption) {
   let jsxElement = null
   const functionBody = path.get('body').get('body')
-  if (functionBody.parent && functionBody.parent.type === 'JSXElement') {
+  if (functionBody.parent &&
+    (functionBody.parent.type === 'JSXElement' || functionBody.parent.type === 'JSXFragment')
+  ) {
     const maybeJsxElement = functionBody.find(c => {
-      return c.type === 'JSXElement'
+      return (c.type === 'JSXElement' || c.type === 'JSXFragment')
```

Comment:
Do we use this jsxElement in such a way later in the code that having it as a JsxFragment would cause an issue?

Reply:
I think should be okay since in `processJSXElement` it always tries to get `openingElement` before `applyAttributes` which will always be undefined for [`JSXFragment`](https://babeljs.io/docs/en/babel-types#jsxfragment)
So the fragments won't be annotated, and the `processJSXElement` then checks for `annotateFragments` before recursing. The naming of these seems a bit confusing tho